docs/mpi.rst

```diff
@@ -61,7 +61,7 @@ multiprocessing issues that Open MPI has with RDMA which typically results in se
 does not have noticeable performance impact since most of the heavy communication is done by NCCL, which will use RDMA
 via RoCE or InfiniBand if they're available (see `Horovod on GPU <gpus.rst>`_).  Notable exceptions from this rule are
 models that heavily use ``hvd.broadcast()`` and ``hvd.allgather()`` operations.  To make those operations use RDMA,
-read the `Open MPI with RDMA <#open-mpi-with-rdma>`_ section below.
+read the **Open MPI with RDMA** section below.
```

Comment:
Not nice but works:
```suggestion
read the `Open MPI with RDMA <#id1>`_ section below.
```

Reply:
Great, then let's go with this quick fix (as long as it works).